tests/phpunit/tests/block-template-utils.php

```diff
@@ -161,6 +161,53 @@ public function test_build_block_template_result_from_file() {
 		$this->assertEmpty( $template_part->modified );
 	}
 
+	/**
+	 * @dataProvider data_build_block_template_result_from_file_injects_theme_attribute
```

Comment:
Let's add `@ticket`, `@covers` and `@param` annotations to the test method.

```php
 * @ticket 59325
 *
 * @covers ::_build_block_template_result_from_file
 *
 * @dataProvider data_build_block_template_result_from_file_injects_theme_attribute
 *
 * @param string $filename         The template's filename.
 * @param string $expected_content The expected block markup.
```

Does `$expected_content` make sense here or should it be `$expected_markup` or `$expected`? No big deal, happy to keep as-is if it makes sense in context.

Reply:
@costdev you bit me on this 😓

Reply:
Ah, you're right, `$expected_content` isn't that great. I think I chose it since we're comparing to `$template->content`, but it doesn't make too much sense as an argument name. I'll go with one of your suggestions!

Reply:
Fixed in 779f10728b4f049d61068c80eacedb5fa1e1a55a and e3e00f6ffc9c5d3f9edf80087b576e880ff7daa4.